target/linux/lantiq/base-files/etc/board.d/02_network

```diff
@@ -183,4 +190,3 @@ ucidef_set_interface_wan "$interface_wan" "pppoe"
 board_config_flush
 
 exit 0
-
```

Comment:
no it isn't a wanted feature. but that change is unrelated to your commit and should be fixed a separate (cleanup) commit.